client/me/security-2fa-sms-settings/index.jsx

```diff
@@ -89,44 +77,45 @@ const Security2faSMSSettings = createReactClass( {
 		}
 
 		return false;
-	},
+	}
 
-	onVerifyByApp: function ( event ) {
+	onVerifyByApp( event ) {
 		event.preventDefault();
 		this.verifyByApp = true;
 		this.submitSMSSettings();
-	},
+	}
 
-	onVerifyBySMS: function ( event ) {
+	onVerifyBySMS( event ) {
```

Comment:
Should we make this an arrow function so `this` is automatically bound?

Reply:
Done via https://github.com/Automattic/wp-calypso/pull/49000/commits/f8c03b7fffc7fce1996b2b56526614421eb319c0